consul_exporter.go

```diff
@@ -107,7 +107,7 @@ type promHTTPLogger struct {
 }
 
 func (l promHTTPLogger) Println(v ...interface{}) {
-	level.Error(l.logger).Log("msg", v)
+	level.Error(l.logger).Log("msg", fmt.Sprintf("%v", v))
```

Comment:
Sorry my previous comment was confusing, it should be `level.Error(l.logger).Log("msg", fmt.Sprint(v))`